### PR TITLE
Define a custom .env when running bashunit

### PR DESCRIPTION
**Pull request by Chemaclass**
## 📚 Description

Issue: https://github.com/TypedDevs/bashunit/issues/187

## 🔖 Changes

- Add `--env` option
- Create and use `.env.test` in all acceptance tests

## ✅ To-do list

- [x] Make sure that all the pipeline passes
- [x] Make sure you have updated the documentation to reflect the changes or new features
- [x] Make sure to update the `CHANGELOG.md` to reflect the new feature or fix

### CODE REVIEW

File: .env.test [hunk 1]
```diff
@@ -0,0 +1,3 @@
+SHOW_HEADER=false
+HEADER_ASCII_ART=false
+SIMPLE_OUTPUT=false
```
Comment (khru) [inline .env.test:3]: If I remember correctly, I think we commented that this .env should be in the acceptance test folder.

File: bashunit [hunk 2]
```diff
@@ -40,6 +40,12 @@ while [[ $# -gt 0 ]]; do
       SIMPLE_OUTPUT=false
       shift
       ;;
+    --env)
+      # shellcheck disable=SC1090
+      source "$2"
+      shift
+      shift
+      ;;
```
Comment (khru) [inline bashunit:48]: why 2 `shift`?

File: tests/acceptance/bashunit_test.sh [hunk 3]
```diff
@@ -109,9 +109,9 @@ function test_bashunit_should_allow_test_drive_development() {
 
   set +e
 
-  assert_contains "$fixture_start" "$(./bashunit --verbose "$test_file")"
-  assert_contains "$fixture_end" "$(./bashunit --verbose "$test_file")"
-  assert_general_error "$(./bashunit --verbose "$test_file")"
+  assert_contains "$fixture_start" "$(./bashunit --env .env.test "$test_file")"
+  assert_contains "$fixture_end" "$(./bashunit --env .env.test "$test_file")"
+  assert_general_error "$(./bashunit --env .env.test "$test_file")"
```
Comment (khru) [inline tests/acceptance/bashunit_test.sh]: should the path to the `.env` file be on a variable an on a beforeEach?

File: bashunit (hunk 2 above)
Comment (Chemaclass) [reply bashunit:48]: 1st) for the -e|--env
2nd) for the value itself
> **Note**: You can see the filter using the same strategy.

**Review by khru — APPROVED** [review]

**Review by fabriziofs — APPROVED** [review]

**Pull request merged**